toolkit/tools/pkg/imagecustomizerlib/liveosisobuilder.go

```diff
@@ -705,7 +705,7 @@ func (b *LiveOSIsoBuilder) generateInitrdImage(rootfsSourceDir, artifactsSourceD
 		return fmt.Errorf("failed to initialize chroot object for %s:\n%w", rootfsSourceDir, err)
 	}
 
-	requiredRpms := []string{"squashfs-tools", "tar", "device-mapper"}
+	requiredRpms := []string{"squashfs-tools", "tar", "device-mapper", "curl"}
```

Comment:
You need to update these files as well:
- /toolkit/tools/pkg/imagecustomizerlib/testdata/iso-files-and-args-config.yaml
- /toolkit/tools/pkg/imagecustomizerlib/testdata/iso-os-prereqs-config.yaml

It might also be worth running the ISO tests:

`sudo go test -run '^TestCustomizeImageLiveCd' ./pkg/imagecustomizerlib -args --base-image-core-efi <core-efi.vhdx>`

Reply:
thx - fixed.